src/PushManager.ts

```diff
@@ -32,17 +32,15 @@ let serviceWorkerRegistration = null
  * Push Manager handles the registration and subscription for web push.
  */
 export default class PushManager {
-  private static createRequest: (action: string, args: ArgsBuilder, options: any) => void
-
-  static setCreateRequest(value: (action: string, args: ArgsBuilder, options: any) => void): void {
-    PushManager.createRequest = value
-  }
+  public constructor(
+    private createRequest: (action: string, args: ArgsBuilder, options: any) => void
```

Comment:
nit: this type is used in several files, it would be a good thing to extract it to a common file

Reply:
Thanks, will do it as part of the refactoring that I'm currently working on.